### ISSUE TITLE
[Crash]: Reinforced Source Input Hatch Crashed on Assigning it to Take from an Ender Source Jar

### ISSUE THREAD

**Issue by SquirrelCMDR** [AllTheMods/ATM-10#1377]
### Possible Fixes

Yes

### Modpack Version

2.1

### What happened?

Noticed the Source input could be assigned to take or send Source.

Placed an Ender Source Jar on top of the Reinforced Source Hatch.
Used the Dominion Wand (from base Ars). Right Clicked the Ender Source Jar, then Shift Right Clicked the hatch.
Immediate crash.

Currently unable to re-enter the world.

--- I believe it was the reinforced hatch, could hold 2000 source.

EDIT: Was the Reinforce Source Input Hatch

### Crash Log

https://gist.github.com/SquirrelCMDR/2a13260cc1252ba27dec04249150ae81

**Comment by SquirrelCMDR** [comment]
Another note, this hatch was placed on a Runic Crucible multiblock and it was properly formed (the machine parts all turned black like they are supposed to).

**Comment by alekthefirst** [comment]
Yes its currently unusable and ATM star thus uncraftable, i believe it along with some other MMR issues will be fixed in the next patch

**Comment by DaddyLongspoon** [comment]
I know this doesn't solve your problem but I have the runic crucible working (it's the only one of them for me). im feeding the source input from ae2 exporter, so I have the ars energestique disk storing some source, suspect maybe the source hatch is intended to work with that and perhaps doesn't interact with the dominion wand..? (ie whatever the 'digital source'  that ars energestique enables ae2 to put to disk. It'll probably confound a lot of people if this is the case, perhaps a me source disk should be part of the quest progression..
